documentation/logs.md

```diff
@@ -176,7 +176,14 @@ Verbose output is useful when you want to see the results for Bolt actions on yo
 usually not printed to standard out (stdout). Verbose isn't a log level, but is a way of telling
 Bolt to output additional information in a human-readable format. Verbose output is particularly
 useful for debugging your tasks and plans - if you're not sure why something is failing, try running
-it with `--verbose` to get more information.
+it with `--verbose` to get more information. You can also use `out::verbose` in plans to print messages. For example:
+
+ ```shell
+plan mymodule::myplan {
+  out::verbose('Hello world')
+}
+  ```
+Would print “Hello world” only when run in verbose mode.
```

Comment:
Not too sure about this.

Reply:
Looking at this doc again, I think we need to rework this section a little bit. Could you move the following sentences to the top of the section (below the heading "Using `verbose output`):
> Verbose output is useful when you want to see the results for Bolt actions on your targets that are usually not printed to standard out (stdout). Verbose isn't a log level, but is a way of telling Bolt to output additional information in a human-readable format.

You can leave the rest (Verbose output is particularly useful for debugging your tasks and plans...) where it is.  Let me know if this isn't clear. I wish Git would let me add comments to the collapsed sections.